ross/rotor_assembly.py

```diff
@@ -1312,6 +1325,238 @@ def load(file_name):
         with open(file_name, 'rb') as f:
             return pickle.load(f)
 
+    @classmethod
+    def from_section(cls,
+                     leng_data=list,

```

Comment:
I did this on purpose to force the input to be a list. The shaft sections are all dealt as a list of shaft section' length, -outer diameter and -inner diameter along the code. If each sections has a different values for these inputs, the elements will have different lengths and diameters, although the number of elements per section is always the same (we may change it later).

Reply:
I understand now, but this won't work. For example, if you try:

```python
def foo(a=list):
    print(a)

>>> foo(a=1)
1
```
In this case, an `int` was given instead of a `list` and the code worked anyways. 
When you use `def foo(a=list)` you are only setting the default argument for `a` as the object `list`, so it does not work as in a statically typed language.

I think you could check for the type of the argument throughout the code. If you are using `len(argument)` you are already checking if the argument is an iterable, so there is no need to additional checks for that on the code.

Reply:
I see now. You are right. So, I'm going to remove this "list" default setting and leaving it without any default options. Anyway, the user should input the shaft sections to the correct operation of the code